### PR TITLE
Register fixes

### PR DESCRIPTION
**Pull request by ilzeLourens**

### CODE REVIEW

File: skietbaan_fe/src/components/ScoreCapture.js [hunk 1]
```diff
@@ -175,317 +199,291 @@ export default class search extends Component {
...
-                  onClick={() => this.CameraClicked()} alt=''></img>
               </div>
-              <label className="labelIcon">Capture score</label>
-            </div>
-            <div className={(this.state.showCamera && !this.state.ImageTaken)
-              || this.state.ImageTaken ? "hidden" : "submit-button-elements"}>
-              <div className="button-hover ">
-                <img src={graySubmit} onClick={() => this.GetLocation()}
-                  className="button-that-submits" alt=''></img>
+              <div className={this.state.scoreSaved ? "sucess-container" : "hidden"}>
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ScoreCapture.js]: inconsistent naming convention fro css classes

File: skietbaan_fe/src/components/ScoreCapture.js [hunk 2]
```diff
@@ -175,317 +199,291 @@ export default class search extends Component {
...
+              <div className={this.state.ImageTaken || this.state.showCamera
+                ? "hidden" : "submit-button-elements"}>
+                <div className="button-hover">
+                  <img src={cameraGray}
+                    id="btnScoreCapture" className="btnScoreCapture"
+                    onClick={() => this.CameraClicked()} alt=''></img>
+                </div>
+                <label className="labelIcon">Capture score</label>
+              </div>
+              <div className={(this.state.showCamera && !this.state.ImageTaken)
```
[the remark this answers is not in the document]
Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: fixed

File: skietbaan_fe/src/components/ScoreCapture.js [hunk 3]
```diff
@@ -175,317 +199,291 @@ export default class search extends Component {
...
                 </div>
                 <label className="labelIcon">Submit</label>
               </div>
-              <div className={this.state.ImageTaken ? "submit-button-elements third float-right" : "hidden"} >
-                <div className="button-hover">
-                  <img src={grayRetry}
-                    id="btnScoreCapture" className="retake" onClick={() => this.RetakePhoto()}
-                    alt=''>
-                  </img>
+              <div className="icon-pushdown no-margin">
```
[the remark this answers is not in the document]
Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: fixed

File: skietbaan_fe/src/components/ScoreCapture.js (hunk 1 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: what? its the same

File: skietbaan_fe/src/components/Login.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ import { validateUsername } from './Validators.js';
 import { getCookie } from './cookie.js';
 import { URL } from '../actions/types.js';
 import back from '../components/assets/Back.png';
-import skietbaan from '../components/assets/skietbaan.png';
+import skietbaan from '../components/assets/biggerskietbaan.png';
```
[the remark this answers is not in the document]
Comment (ilzeLourens) [reply skietbaan_fe/src/components/Login.js]: fixed

File: skietbaan_fe/src/components/ScoreCapture.js (hunk 1 above)
Comment (FredJEhlers) [reply skietbaan_fe/src/components/ScoreCapture.js]: Apologise confused state var with css classname

**Pull request merged**